pkg/machine/config.go

```diff
@@ -73,6 +73,14 @@ type StartOptions struct {
 	Rosetta bool
 }
 
+type CpOptions struct {
```

Comment:
if this struct is only ever used in cmd/podman it would make sense to declare there as private so we do not have to carry it as "public" exposed API.